### PR TITLE
Actions should trigger jQuery event on the body

### LINKED ISSUE #18: JS notification manager
We need something like the php notification manager in JS.
The problem is that when you accept a share via the notifications,
the share popup on the file list should be closed and the file list updated.

Also when accepting the the share in the popup the notification should be removed (not only after the next polling).

### PR DESCRIPTION
**Pull request by nickvergessen**
Fix #18

### CODE REVIEW

**Comment by nickvergessen** [comment]
@PVince81

File: js/app.js [hunk 1]
```diff
@@ -107,6 +107,13 @@
                 type: actionType,
                 success: function(data) {
                     self._removeNotification($notification.attr('data-id'));
+					$('body').trigger(new $.Event('OCA.Notification.Action', {
+						notification: self.notifications[$notification.attr('data-id')],
+						action: {
+							url: actionUrl,
+							type: actionType
+						}
+					}));
```
Comment (PVince81) [inline js/app.js]: weird indent

**Comment by PVince81** [comment]
Apart from the weird indent, code looks good :+1:

File: js/app.js (hunk 1 above)
Comment (nickvergessen) [inline js/app.js]: yeah this is because everything else is spaces,
I will change "everything else" soon

**Comment by nickvergessen** [comment]
Fixed, https://github.com/owncloud/notifications/pull/62/files?w=1 is better for PR review now.

**Pull request merged**